api/core/model_runtime/model_providers/bedrock/text_embedding/cohere.embed-multilingual-v3.yaml

```diff
@@ -0,0 +1,8 @@
+model: cohere.embed-multilingual-v3
+model_type: text-embedding
+model_properties:
+  context_size: 512
```

Comment:
https://github.com/langgenius/dify/blob/a18dde9b0dda6a32d87903c00da41e5211a5901c/api/core/model_runtime/model_providers/cohere/text_embedding/embed-multilingual-v3.0.yaml#L4

See this. `max_chunks`